### PR TITLE
Second go, entity category created and mapped

### PR DESCRIPTION
**Pull request by TheMcKenzie**
This time everything should be right

### CODE REVIEW

File: src/main/java/org/wwarn/vivax/manager/domain/Category.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import java.util.Objects;
+
+/**
+ * A Category.
+ */
+@Entity
+@Table(name = "category")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@Document(indexName = "category")
+public class Category implements Serializable {
```
Comment (ignacioSuay) [inline src/main/java/org/wwarn/vivax/manager/domain/Category.java]: you need to Add the @Audited

Comment (TheMcKenzie) [inline src/main/java/org/wwarn/vivax/manager/domain/Category.java]: [image: Inline image 1]

  It's there in my version... How is that possible? It's the first thing I
put on the file...

File: src/main/java/org/wwarn/vivax/manager/domain/Category.java [hunk 2]
```diff
@@ -14,15 +15,23 @@
 /**
  * A Category.
  */
+@Audited
 @Entity
-@Table(name = "category")
+@Table(name = "Category")
 @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
 @Document(indexName = "category")
 public class Category implements Serializable {
```
Comment (ignacioSuay) [inline src/main/java/org/wwarn/vivax/manager/domain/Category.java]: still need to add @Audited

File: src/main/java/org/wwarn/vivax/manager/domain/Category.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+@Audited
+@Entity
+@Table(name = "Category")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@Document(indexName = "category")
+public class Category implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+    
+    @Version
```
Comment (ignacioSuay) [inline src/main/java/org/wwarn/vivax/manager/domain/Category.java]: Please try to put the version field the last one, to improve code rediability

**Pull request merged**